examples/carousel/image/index.html

```diff
@@ -44,10 +45,11 @@
             width: 50px;
             height: 50px;
             border-radius: 50%;
+            border: 1px solid;
             align-self: center;
             position: absolute;
             z-index: 1;
-            transition: background-color 300ms, border-color 300ms, color 300ms;
+            transition: background-color 300ms, color 300ms;
```

Comment:
```suggestion
            transition: background-color 300ms, border-color 300ms, color 300ms;
```
restore border-color animation